thyrel-web/src/hooks/SessionProvider.tsx

```diff
@@ -27,6 +30,31 @@ export function SessionContextProvider({
   const { websocket } = useWebsocketContext();
   const { enqueueSnackbar } = useSnackbar();
 
+  console.log(currentElement);
+  function onSave(content: string) {
+    const elementId = currentElement?.id;
+    const url = `element/${elementId}`;
+
+    const elementContent =
+      currentElement?.type == 1 ? { drawimage: content } : { text: content };
+
+    client<HolyElement>(url, {
```

Comment:
c'est pas feux mais je préf comme ça (et supp la variable url en haut)
```suggestion
    client<HolyElement>(`element/${elementId}`, {
```